spec/classes/puppet_profile_master_spec.rb

```diff
@@ -15,6 +15,7 @@
         {
           "operatingsystem" => "CentOS",
           "operatingsystemrelease" => [
+            "6",
```

Comment:
I definitely haven't tested this class on CentOS 6.
Can you confirm that it actually work on CentOS 6?